lucene/CHANGES.txt

```diff
@@ -25,6 +25,8 @@ API Changes
 * LUCENE-8811: Deprecated BooleanQuery#setMaxClauseCount() and getMaxClauseCount
   have been removed (Atri Sharma, Adrien Grand, Alan Woodward)
 
+* LUCENE-8905: Better Defence Against Malformed Arguments in TopDocsCollector
```

Comment:
We don't typically capitalize the whole sentence in `CHANGES.txt`, just the first word.